modules/basegl/src/processors/volumeprocessing/volumenormalizationprocessor.cpp

```diff
@@ -59,19 +59,28 @@ const ProcessorInfo VolumeNormalizationProcessor::processorInfo_{
     "Volume Operation",                         // Category
     CodeState::Stable,                          // Code state
     Tags::GL,                                   // Tags
+    R"(Normalizes the selected channels of the input volume based on its data range to range [0, 1].
+The format of the resulting volume is 32bit floating point matching the number of channels in the input volume.
```

Comment:
you can indent the text since you use _unindentHelp

Reply:
I know.